tools/pyi/gen_pyi.py

```diff
@@ -1101,6 +1101,7 @@ def replace_special_case(hint: str) -> str:
                     )
                 )
             ],
+            "__contains__": ["def __contains__(self, other: Any) -> _bool: ..."],
```

Comment:
This is doubled, considering that the type hint in [torch/_tensor.py](https://github.com/pytorch/pytorch/pull/129653/files#diff-41eb33210e32f7a565a420a78573740a643b5f124726e7013e8b96aad158948e) should take priority, but some type checkers might ignore it when they detect the pyi file.